src/sssom/util.py

```diff
@@ -423,7 +443,7 @@ def filter_redundant_rows(df: pd.DataFrame, ignore_predicate: bool = False) -> p
     else:
         key = [SUBJECT_ID, OBJECT_ID, PREDICATE_ID]
     dfmax: pd.DataFrame
-    dfmax = df.groupby(key, as_index=False)[CONFIDENCE].apply(max).drop_duplicates()
+    dfmax = df.groupby(key, as_index=False)[CONFIDENCE].apply(np.maximum.reduce).drop_duplicates()
```

Comment:
Why this change? Is `max` (I assume a built-in like `None`, since it is not imported anywhere?) deprecated?

Reply:
Another warning from `pandas` I think

Reply:
Can you link the docs you used to inform the switch to np.reduce? I have not been able to find it googling..

Reply:
Weird ... I rolled back to `max` and the deprecation does not show up. Well, this change has been undone.